services/py-api/src/server/routes/participants_routes.py

```diff
@@ -0,0 +1,49 @@
+from fastapi import APIRouter, Depends
+from starlette.responses import Response
+
+from src.database.db_manager import DB_MANAGER, PARTICIPANTS_COLLECTION, TEAMS_COLLECTION
+from src.database.repository.participants_repository import ParticipantsRepository
+from src.database.repository.teams_repository import TeamsRepository
+from src.database.transaction_manager import TransactionManager
+from src.server.handlers.participants_handlers import ParticipantHandlers
+from src.server.schemas.request_schemas.schemas import ParticipantRequestBody
+from src.server.schemas.response_schemas.schemas import AdminParticipantRegisteredResponse, ErrResponse
+from src.service.participants_service import ParticipantService
+
+participants_router = APIRouter(prefix="/hackathon/participants")
+
+
+# https://fastapi.tiangolo.com/tutorial/dependencies/sub-dependencies/
+# Dependency wiring
+
+
+def _p_repo(db_manager: DB_MANAGER) -> ParticipantsRepository:
```

Comment:
Help me understand why you need these methods here.

Reply:
There is a comment pointing to the docs. This is how Dependency injection is done in FastAPI. I know it's weird, I was also confused the first time.

Reply:
Okay. Got it.